docs/install_apt_debian.md

```diff
@@ -4,10 +4,11 @@ title: "Installation: Debian"
 
 # Debian Installation
 
-Caffe packages are available for `Debian/unstable`. Debian/stable users
-should take a look at Ubuntu installation instruction.  
+Caffe packages are available for `Debian/unstable` (Debian 10). Debian/stable
```

Comment:
Stretch is `Debian/testing` (https://wiki.debian.org/DebianTesting & https://wiki.debian.org/DebianReleases#Current_Releases.2FRepositories), no? Debian/unstable will have the codename "buster", after it gets promoted from being unstable/sid. If the codenames are given later on, I think it is fine to leave it off.